articles/exe-install-scripts.md

```diff
@@ -0,0 +1,334 @@
+# Windows EXE install scripts
+
+## What are EXE install scripts?
+
+EXE install scripts are a way to install software on Windows. EXE installers, such as `Figma-124.3.2.exe`, are self-contained packages with all the files and instructions needed to install software on a Windows device. EXE installers are fully customizable and do not follow the same installation process as MSI installers.
+
+For EXE installers, there is no unique script or command that will work for all installers. MSI installers are typically preferred over EXE installers because they provide a standardized installation process, easier silent deployment, and better integration with Windows Installer Service. If available, MSI installers offer more predictable results in enterprise environments.
+
+Some EXE installers and uninstallers require additional switches or flags to run silently. Common flags include `/S`, `/q`, `/quiet`, `/silent`, or `--silent`.
+
+## Device-scoped install scripts
+
+The recommended way to install software on Windows devices is to use device-scoped install scripts. These scripts install the software for all users on the device and run the installation process with administrator privileges.
+
+Fleet defaults to a device-scoped install script when you add software using an EXE installer.
+
+## User-scoped install scripts
+
+Some software can only be installed for a specific user. In this case, you can use user-scoped install scripts. The software is installed only for the user currently logged in, and the installation process is run with the user's privileges.
+
+### Example user-scoped install script
+
+The install script creates a scheduled task that will automatically be run as the current (logged-in) user. The EXE installer is copied to a public directory accessible by the user, ensuring that even non-administrator users can run the scheduled task to complete the installation. After the task finishes, the installer and the task are deleted.
+
+The use of scheduled tasks allows the installer to run with user-level permissions, which is especially useful when installing software for non-admin users without requiring administrator credentials at the time of execution.
+
+Since the installation is run by the current user, the script does not output the installer's messages to the console. If you need to see the output, you can modify the script to redirect it to a file and append it to the script output.
```

Comment:
@getvictor, do you think it's worth mentioning how to capture the installer output? Something like: (I'm gleaning this from another PowerShell script I had)

To capture the installer output in a log file, modify the Start-Process section of the script like this:

```powershell
$processOptions = @{
    FilePath = $uninstallCommand
    PassThru = $true
    Wait = $true
    RedirectStandardOutput = 'C:\Path\To\LogFile.log'
}
```